app/controllers/application_controller/ci_processing.rb

```diff
@@ -1058,16 +999,13 @@ def breadcrumb_name(_model)
   # Reconfigure selected VMs
   def reconfigurevms
     assert_privileges(params[:pressed])
-    @edit = {}
-    @edit[:key] = "reconfigure__new"
-    @edit[:current] = {}
-    @edit[:new] = {}
+    session[:req_id] = nil
     # check to see if coming from show_list or drilled into vms from another CI
     rec_cls = "vm"
     recs = []
     # if coming in to edit from miq_request list view
     if !session[:checked_items].nil? && (@lastaction == "set_checked_items" || params[:pressed] == "miq_request_edit")
-      @edit[:req_id] = params[:id]
+      session[:req_id] = params[:id]
       recs = session[:checked_items]
```

Comment:
@lgalis It would be preferable if we do not store things in ```session```. If you need a temporary storage then I think an instance variable should be fine in this case.